docs/Basics-Layout.md

```diff
@@ -63,7 +63,7 @@ class AwesomeProject {
   }
 };
 
-AppRegistry.registerComponent('AwesomeProject', () => AwesomeProject);
+AppRegistry.registerComponent('AwesomeProject', () => JustifyContentBasics);
```

Comment:
I can change it. Two things I could do:

1. I can make it so you can override. So right now if you do `'''ReactNativeWebPlayer?runApp=WhateverYouWant` it won't override cause it's hardcoded. But I can have it check if you use a param and only add `runApp=AwesomeProject` if you don't.

2. On the iframe side, I can just duck punch `registerComponent` and not care what is registered.